tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -248,10 +248,10 @@ function kbs_k8s_deploy() {
 
 	echo "::group::Deploy the KBS"
 	if [ "${KATA_HYPERVISOR}" = "qemu-tdx" ]; then
-		echo "Setting up custom PCCS for TDX"
+		echo "Setting up custom Collateral Service for TDX"
 		cat <<- EOF > "${COCO_KBS_DIR}/config/kubernetes/custom_pccs/sgx_default_qcnl.conf"
 {
- "pccs_url": "https://localhost:8081/sgx/certification/v4/",
+ "collateral_service": "https://api.trustedservices.intel.com/sgx/certification/v4/",
```

Comment:
Hi @fidencio !

I'd like to learn more about that part of Intel TDX/SGX attestation; is there a intro public link that you could share with me?

Reply:
https://download.01.org/intel-sgx/sgx-dcap/1.21/linux/docs/ contains a whole lot of documents about DCAP, which is what we use for attestation.